src/test/java/com/nerdapplabs/test/Test_Login.java

```diff
@@ -56,9 +56,11 @@ public boolean LoginFunction() throws IOException {
 	 	WebElement login = driver.findElement( loginBtn);
 	 	//click on login button
 	 	login.click();
-	 	String lgout = driver.findElement(logout).getText();
+		String lgout = driver.findElement(logout).getText();
 	 	String logged = "LOGOUT";
+	 	WebElement Logout= driver.findElement(logout);
 	 	if(lgout.equals(logged)) {
+	 		Logout.click();
```

Comment:
after logout before returning true, must check some page element if control back or not to login page again